mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionPos.scala

```diff
@@ -58,6 +60,7 @@ object CompletionPos:
     val kind =
       if query.isEmpty && !prevIsDot then CompletionKind.Empty
```

Comment:
I believe we should change this, so `import scala.collection.{AbstractMap, @@}` will work

Reply:
Where do we check it? Or do you mean we should check it here?

Reply:
Yes, I mean we should check it here